lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -1818,8 +1818,35 @@ bool TypeSystemSwiftTypeRef::IsImportedType(opaque_compiler_type_t type,
   VALIDATE_AND_RETURN(impl, IsImportedType, type,
                       (ReconstructType(type), nullptr));
 }
+
 CompilerType TypeSystemSwiftTypeRef::GetErrorType() {
-  return m_swift_ast_context->GetErrorType();
+  auto impl = [&]() {
+    using namespace swift::Demangle;
+    Demangler Dem;
+    auto *error_type = Dem.createNode(Node::Kind::Type);
```

Comment:
I think the surrounding functions spell this
`NodePointer error_type = ...`
not a big deal.

Reply:
I noticed a mix of `auto` and `NodePointer` in this file. If we have a convention here, happy to follow it.